test/ParameterStoreBuildWrapperPluginTest.groovy

```diff
@@ -84,5 +125,38 @@ class ParameterStoreBuildWrapperPluginTest {
             assertEquals(ParameterStoreBuildWrapperPlugin.class, result)
         }
     }
+
+    class withGlobalParameter {
```

Comment:
I like this.  This helps to indicate that we're unit testing `withGlobalParameter`.